lineage.cue

```diff
@@ -92,8 +92,8 @@ import (
 		_schemas: [#SchemaDef & {version: [0, 0]}]
 	}
 
-	_forwardLenses: [ for lens in SL if {lens.to[0] > lens.from[0]} {lens}]
-	_backwardLenses: [ for lens in SL if {lens.to[0] <= lens.from[0]} {lens}]
+	_forwardLenses: [ for lens in lenses if {lens.to[0] > lens.from[0]} {lens}]
+	_backwardLenses: [ for lens in lenses if {lens.to[0] <= lens.from[0]} {lens}]
```

Comment:
@sdboyer Do I get it right that this may be removed if the order check will be in Go code now?

Reply:
I only removed the alias from [here](https://github.com/grafana/thema/pull/177/files#diff-331e4a81b037b5b35f4015d6aa015ed24f7d301d8e0491660a8aa3397f89d6b5L82), so it should be the same.

Reply:
these are needed - we're constructing values there which are used in `#Translate`

Reply:
I guess it will depend on how restrictive we define the requirements for lenses definition